polaris-react/src/components/TextField/TextField.scss

```diff
@@ -324,6 +380,20 @@ $spinner-icon-size: 12px;
 .SpinnerIcon {
   height: $spinner-icon-size;
   width: $spinner-icon-size;
+
+  #{$se23} & {
+    height: auto;
+    width: auto;
+    // stylelint-disable-next-line -- se23
+    position: absolute;
```

Comment:
`position: absolute` allows the caret icon (`20px`) to be bigger than the the button its in (`12px`). Can't use height/width since that would resize the arrow and we want it to be its original size, just crop the viewbox